policy-test/tests/api.rs

```diff
@@ -331,3 +319,23 @@ fn mk_admin_server(ns: &str, name: &str) -> k8s::policy::Server {
         },
     }
 }
+
+async fn retry_watch_server(
+    client: &kube::Client,
+    ns: &str,
+    pod_name: &str,
+) -> tonic::Streaming<grpc::inbound::Server> {
+    // Port-forward to the control plane and start watching the pod's admin
+    // server's policy and ensure that the first update uses the default
+    // policy.
+    let mut policy_api = grpc::PolicyClient::port_forwarded(client).await;
+    loop {
+        match policy_api.watch_port(ns, pod_name, 4191).await {
+            Ok(rx) => return rx,
+            Err(error) => {
+                tracing::error!(?error);
```

Comment:
maybe this should also say something like 
```suggestion
                tracing::error!(?error, ns, pod_name, "failed to watch policy for port 4191");
```
or similar? just logging the error message may not make it obvious where in the test the error came from...

alternatively we could `#[instrument]` this function, but i think just saying what we were doing when the error occurred is fine.